modules/executors/variables.tf

```diff
@@ -155,9 +149,3 @@ variable "docker_registry_mirror" {
   default     = ""
   description = "A URL to a docker registry mirror to use (falling back to docker.io)"
 }
-
-variable "assign_public_ip" {
```

Comment:
I vaguely remember that no public IP meant no internet access either but I might be wrong

Reply:
Should be fine because of NAT but would be good to double check. Id be surprised if that was true

Reply:
Do we always have a NAT in our network? I thought we only had it if the executors are in private mode. Maybe we need to tweak that to always be the case?